### PR TITLE
Media local storage

### PR DESCRIPTION
**Pull request by Leepak777**
Added upload file section
will allow upload within five files less than 15mb

when uploaded file, it will be converted into a string added to a string array of files.
Then save to localstorage under memoid directory
fetch the array of files string array with memo id

new component MediaDisplay read the strings and render according to media type

### CODE REVIEW

**Comment by vinn03** [comment]
could you change all occurrences of "blue" tailwind css stylings to "sky" instead?

File: frontend/src/components/CreateMemo/components/MemoForm/memoForm.tsx [hunk 1]
```diff
@@ -117,7 +157,23 @@ const MemoForm = (props:{
             />
             </label>
         </div>
+        <div className="input-container">
+          <label htmlFor="files" className="text-xl text-blue-800">
```
Comment (vinn03) [inline frontend/src/components/CreateMemo/components/MemoForm/memoForm.tsx:161]: here for example

Comment (Leepak777) [reply frontend/src/components/CreateMemo/components/MemoForm/memoForm.tsx:161]: oki

**Comment by vinn03** [comment]
@jerrydngzh thoughts? 

lgtm otherwise

**Review by jerrydngzh — APPROVED** [review]
this is good, realized i didnt specify what data types to upload, thats my bad.

The current backend service i'm writing (or well borrow code from) expects an array of `Files` i.e. sending data as multipart/form-data (not base64). 

I do have a base64 version of this lying around somewhere, but it's not tested, so ill see what I do with it.

The rendering of images is actually just `URL/<uid>/file_name`, as this will be put on a Google Cloud Storage Bucket, and can just be read directly via a url. 

ex:
https://storage.googleapis.com/cmpt474_media_service_store/emoticon-looking-up-raising-hands-600nw-444822673.jpeg

**Pull request merged**